frontend/src/components/maps/leaflet/InfoSlideOut/InfoSlideOut.tsx

```diff
@@ -165,36 +235,42 @@ const InfoControl: React.FC<InfoControlProps> = ({ open, setOpen, onHeaderAction
           <InfoButton
             id="slideOutInfoButton"
             variant="outline-secondary"
-            onClick={setOpen}
+            onClick={() => {
+              if (!open) {
+                setOpen();
+                setParcelInfoOpen(true);
+                setBuildingsOpen(false);
+              } else if (open && !parcelInfoOpen) {
+                setParcelInfoOpen(true);
+                setBuildingsOpen(false);
+              } else {
+                setOpen(); //close the slide out
+              }
+            }}
             className={clsx({ open })}
           >
             <InfoIcon />
           </InfoButton>
         </TooltipWrapper>
-        <InfoMain className={clsx({ open })}>
-          {popUpContext.propertyInfo ? (
-            <>
-              <HeaderActions
-                propertyInfo={popUpContext.propertyInfo}
-                propertyTypeId={popUpContext.propertyTypeID}
-                onLinkClick={onHeaderActionClick}
-                jumpToView={jumpToView}
-                zoomToView={zoomToView}
-                canViewDetails={canViewProperty}
-                canEditDetails={canEditProperty}
-              />
-              <InfoContent
-                propertyInfo={popUpContext.propertyInfo}
-                propertyTypeId={popUpContext.propertyTypeID}
-                addAssociatedBuildingLink={addAssociatedBuildingLink}
-                canViewDetails={canViewProperty}
-                canEditDetails={canEditProperty}
-              />
-            </>
-          ) : (
-            <p>Click a pin to view the property details</p>
+        {open &&
+          popUpContext.propertyInfo &&
+          canViewProperty &&
+          popUpContext.propertyTypeID === PropertyTypes.PARCEL && (
```

Comment:
Only show the button when the slide out is open on a parcel, and the user has permissions to view all details